service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/NonConsultationResourceMapper.java

```diff
@@ -254,4 +278,18 @@ private boolean isIgnoredResource(Resource resource) {
 
         return false;
     }
+
+    private boolean isEndedAllergyList(Resource resource) {
+        if (resource.getResourceType().equals(ResourceType.List)) {
+            var list = (ListResource) resource;
+            var endedAllergies = Optional.ofNullable(list.getCode())
+                    .map(CodeableConcept::getCodingFirstRep)
+                    .filter(coding -> coding.getCode().equals(ENDED_ALLERGIES_CODE))
```

Comment:
Sorry Rory I forgot about a small thing here, could you switch order of ENDED_ALLERGIES_CODE and coding.getCode() to `ENDED_ALLERGIES_CODE.equals(coding.getCode())`? In case getCode returns null, it won't be a problem.

Reply:
Good spot, added this aswell thanks :)